### DISCUSSION TITLE
Running Scripts from My Files

### DISCUSSION THREAD

**Discussion by primetecabt** [Ylianst/MeshCentral#6361] [Issues]
I would like to know if MeshCentral can run the scripts saved in my files from the terminal tab, as I can run them through the general tab or through remote access.

![image1](https://github.com/user-attachments/assets/e7128937-e77b-48a6-9e31-f47f23d4a2a9)
![image2](https://github.com/user-attachments/assets/bb4d7932-c1ec-4c8e-bb74-946ba2bb8edf)

**Comment by si458** [comment]
Yes, if u just select which one u want to run and then run it, u then check the console tab for its output

Remembering the scripts are not interactive!

So u will see its output in the console, but u can not interact with the scripts

**Reply by primetecabt** [reply]
Sorry for the delay in responding, but after a few attempts I managed to complete the process as explained. Thanks for the support.
